src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py

```diff
@@ -42,6 +42,7 @@
 class ContainerType(Enum):
     PythonLTS = 0
     CustomContainer = 2
+    NodeLTS = 3
```

Comment:
please also update _params.py for container_type. Can you use ContainerType as enum value to avoid duplicated work?